VTUL/iawa
# LIBTD-1820: Remove changeLayout button in Mirador Viewer

Completely remove the top menu as only the fullscreen button left after removing the layout button. The fullscreen function still exists in the main menu.

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1820) (:star:)

# What does this Pull Request do? (:star:)
This PR removes top menu which includes changeLayout button and fullscreen button from Mirador viewer. The fullscreen function still exists on the top right corner. 

# What's the changes? (:star:)

* Fix formatting in app/services/hyrax/collection_thumbnail_path_service_override.rb (unrelated)
* Remove top menu from Mirador viewer

# How should this be tested?

* Bring up the application.  
* Run as railsapps and bundle exec rake iawa:batch_import["Ms1990_025_Rudoff.index.csv"] to get some items with image files
* Should no longer see top menu at the top of Mirador Viewer
![Mirador_viewer_top_menu](https://user-images.githubusercontent.com/8228221/61755428-71063a80-ad85-11e9-840a-b0fad4dd4c9f.png)


# Interested parties
Tag (@pmather) interested parties

(:star:) Required fields
